<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/line-chart.png</normaloff>../Icons/line-chart.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="generateButton">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="firstValuecomboBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="secondValuecomboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="CompareValues">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Compare values</string>
   </property>
  </widget>
  <widget class="QLabel" name="XAxis">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X-Axis</string>
   </property>
  </widget>
  <widget class="QComboBox" name="xAxiscomboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
